### DISCUSSION TITLE
1.20.x javadoc

### DISCUSSION THREAD

**Discussion by vico93** [cc-tweaked/CC-Tweaked#1499] [General]
Is there a link to the current version javadoc?

I want to send [this part](https://tweaked.cc/mc-1.19.x/javadoc/dan200/computercraft/api/turtle/TurtleUpgradeSerialiser.html) but updated to a modder to see if he have interest in doing some addon mods for CC:T, but i dont know if something changed between 1.19 and 1.20 versions of CC:T...

**Comment by SquidDev** [comment]
We currently only publish docs for 1.19, but nothing changed between the two.

**Reply by vico93** [reply]
Thanks for the clarification!
